### PR TITLE
2024 districs

### PR DESCRIPTION
**Pull request by asherbolotski**
this is 2024 districts branch that never got put into main

### CODE REVIEW

File: ted/src/main/java/frc/robot/commands/RewindFeederCommand.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.common.FeederMode;
+import frc.robot.control.Constants;
+import frc.robot.subsystems.FeederSubsystem;
+
+/**
+ * Forms a command to feed the note to the shooter or dunker
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/commands/RewindFeederCommand.java]: Fix this comment

File: ted/src/main/java/frc/robot/commands/RewindFeederCommand.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  public void initialize()
+  {
+    // set the feeder in the right direction 
+    feeder.setFeederMode(direction);
+    // intentionally *not* setting feeder speed to 0 here. 
+    // If the feeder is currently running, don't stop it. 
+    timer.reset();
+    timer.start();
+    done = false;
+    System.out.println("Starting FeedNoteCommand");
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/commands/RewindFeederCommand.java]: fix print statement.

File: ted/src/main/java/frc/robot/control/AutonomousChooser.java [hunk 3]
```diff
@@ -121,7 +126,7 @@ public Command getCommand(){
         subsystems.getCameraSubsystem().setBotPoseSource();
         return new ParallelCommandGroup(
             new ShooterSpinUpAutoCommand(subsystems.getShooterOutfeedSubsystem()),
-            new UseFusedVisionInAutoCommand(subsystems.getDriveTrainSubsystem()),
+            //new UseFusedVisionInAutoCommand(subsystems.getDriveTrainSubsystem()),
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/control/AutonomousChooser.java]: make a flag in constants.java useFusedVisionInAuto, set it to true and change this line to use the flag.

File: ted/src/main/java/frc/robot/subsystems/ClimberSubsystem.java [hunk 4]
```diff
@@ -250,7 +250,7 @@ public void moveClimbersToSensorReset() {
     public void periodic() {
         // incremenet the counter
         ++counter;
-
+
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/subsystems/ClimberSubsystem.java]: remove added whitespace

**Review by josieammer — CHANGES_REQUESTED** [review]
just a few small clean-ups

File: ted/src/main/java/frc/robot/subsystems/ClimberSubsystem.java (hunk 4 above)
Comment (asherbolotski) [reply ted/src/main/java/frc/robot/subsystems/ClimberSubsystem.java]: removed

File: ted/src/main/java/frc/robot/control/AutonomousChooser.java (hunk 3 above)
Comment (asherbolotski) [reply ted/src/main/java/frc/robot/control/AutonomousChooser.java]: added

File: ted/src/main/java/frc/robot/commands/RewindFeederCommand.java (hunk 1 above)
Comment (asherbolotski) [reply ted/src/main/java/frc/robot/commands/RewindFeederCommand.java]: fixed

File: ted/src/main/java/frc/robot/commands/RewindFeederCommand.java (hunk 2 above)
Comment (asherbolotski) [reply ted/src/main/java/frc/robot/commands/RewindFeederCommand.java]: fixed

**Review by josieammer — APPROVED** [review]

**Pull request merged**